qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -190,14 +182,35 @@ def _sample_sequences(self) -> List[Sequence[SequenceElementType]]:
 
         return sequences
 
+    def _get_basis_gates(self) -> Optional[Tuple[str]]:
+        """Get sorted basis gates to use in basis transformation during circuit generation.
+
+        Returns:
+            Sorted basis gate names.
+        """
+        # Basis gates to use in basis transformation during circuit generation for 1Q/2Q cases
+        basis_gates = self.transpile_options.get("basis_gates", None)
+        if not basis_gates and self.backend:
+            if isinstance(self.backend, BackendV2):
+                basis_gates = self.backend.operation_names
```

Comment:
Addressed in 2a6ead4. I've thought around this and ended up with using `Target.get_non_global_operation_names(strict_direction=True)` and excluding them from `operation_names` in order to return "global basis gates" here.
I've stacked which qargs we should give for the argument of `operation_names_for_qargs` in 2Q case, (0, 1) or (1, 0) or both. I noticed that the answer depends on which gates each transpiled circuit uses, but this function is intended to be called before transpilation. So we may have to be a bit conservative here and returning the global basis gates makes sense to me (without taking into account qargs, i.e. physical qubits). That means when users want to use a non-global operation in transpilation, they should set their own `basis_gates` to `transpile_options` (and go to the ordinary `transpile` path in `_transpiled_circuits`) for now. In the future, it would be better that they could specify their own "basis translator" in circuit generation (current RB classes have no such option, though).

Reply:
This sounds good to me. You should inline comment since new behavior is bit tricky.

Reply:
Done in 25b09a9